common/src/View/MapDocument.cpp

```diff
@@ -2475,8 +2475,11 @@ namespace TrenchBroom {
         }
 
         void MapDocument::reloadEntityDefinitions() {
-            auto oldSpec = entityDefinitionFile();
-            setEntityDefinitionFile(oldSpec);
+            const std::vector<Model::Node*> nodes(1, m_world.get());
```

Comment:
How about `const auto nodes = std::vector<Model::Node>{m_world.get()}`? That's the style I've been settling on lately.